### PR TITLE
Team create: prompt for which team email address to use

### PR DESCRIPTION
**Pull request by idrysdale**
screenshot 2019-02-21 at 12 29 38

### CODE REVIEW

File: fk/teamcreate.go [hunk 1]
```diff
@@ -94,3 +136,32 @@ func validateTeamName(teamName string) (string, error) {
...
+		prompter := interactiveYesNoPrompter{}
+		if prompter.promptYesNo("Is this your team email?", "y", nil) {
+			return &onlyKey
+		}
+		return nil
+	} else {
+		invalidEntry := fmt.Sprintf("Please select between 1 and %v.\n", len(keys))
+		for validInput := false; !validInput; {
+			rangePrompt := colour.Info(fmt.Sprintf("[1-%v]", len(keys)))
+			input := promptForInput(fmt.Sprintf(promptWhichKeyFromGPG + " " + rangePrompt + " "))
```
Comment (paulfurley) [inline fk/teamcreate.go]: `promptWhichKeyFromGPG` copy/paste error

File: fk/teamcreate.go [hunk 2]
```diff
@@ -35,13 +38,51 @@ func teamCreate() exitCode {
...
 
+	printHeader("Which is your team email address?")
+
+	keys, err := loadPgpKeys()
+	if err != nil {
+		out.Print(ui.FormatFailure("Error loading pgp keys", nil, err))
+		return 1
+	}
+
+	if len(keys) == 0 {
```
Comment (paulfurley) [inline fk/teamcreate.go]: nice

**Comment by paulfurley** [comment]
Looks great apart from the prompt messages, as agreed in the CLI doc, should be:

> "Which is your team email? [1-2] _"

or

> "Is this your team email? [Y/n]"

**Comment by idrysdale** [comment]
Taken those comments onboard and force pushed an update. Thanks! 👍

**Pull request merged**